client/src/templates/Challenges/classic/editor-tabs.tsx

```diff
@@ -47,7 +47,7 @@ class EditorTabs extends Component<EditorTabsProps> {
               data-cy={`editor-tab-${challengeFile.fileKey}`}
               onClick={() => toggleVisibleEditor(challengeFile.fileKey)}
             >
-              {`${challengeFile.name}.${challengeFile.ext}`}{' '}
+              <span>{`${challengeFile.name}.${challengeFile.ext}`}</span>{' '}
               <span className='sr-only'>editor</span>
```

Comment:
Although outside the scope of the PR, can you add this as translatable string? 🥺 

Editor is translated on other language `reset-warn`, so to stay consistent with the translation.